### ISSUE TITLE
scatterplot-layer does not work with deck.gl@4.1.0 on Safari.

### ISSUE THREAD

**Issue by sideroad** [uber/deck.gl#822]
After I upgrade deck.gl version from 4.0.3 to 4.1.0, Safari does not work.
Here is a situation details.
- deck.gl@4.0.3 and 4.1.0 on Chrome is work well
- deck.gl@4.0.3 on Safari was work well
Here is a code to use deck.gl
https://github.com/sideroad/floras/blob/master/src/components/WorldMap.jsx

## Working example
https://floras.herokuapp.com/en

## Error details
Safari throw error in scatterplot-layer 
https://github.com/uber/deck.gl/blob/master/src/layers/core/scatterplot-layer/scatterplot-layer.js#L104
because of this.state.model is null.

But when I open site on Safari below, no error happen...
https://uber.github.io/deck.gl/#/examples/core-layers/scatterplot-layer
https://uber.github.io/deck.gl/#/documentation/layer-catalog/scatterplot-layer

If there is any good way to debug my code, plz tell me.

**Comment by sideroad** [comment]
I've tested example code which are exists on current deck.gl master branch.
Same error has occurred.
<img width="1193" alt="screenshot" src="https://user-images.githubusercontent.com/411486/29122382-128f64e6-7d4d-11e7-991d-3cb8baa2df08.png">

**Comment by sideroad** [comment]
Safari version is 10.1.2 (12603.3.8)

**Comment by shaojingli** [comment]
@1chandu this is the safari bug I mentioned to you before the release. Safari does not follow WebGL specs which makes model creation failed.

**Comment by ibgreen** [comment]
@shaojingli I believe @1chandu  fixed this bug, if so we should publish a 4.1.1 patch release

**Comment by shaojingli** [comment]
@1chandu Let's release a v4.1.1 patch release for deck.gl if this has been fixed

**Comment by sideroad** [comment]
thanks you guys!
Im looking forward about it!

**Comment by ibgreen** [comment]
I took a second look. Looks like the fix was done in [luma.gl](https://github.com/uber/luma.gl/pull/294) and has already been published in the v4.0.1 patch release of luma.  @sideroad let us know if bumping luma works for you.

To make sure installers picks up the luma patch, we might want update deck.gl's package.json and increase the minimum luma.gl dependency to 4.0.1.

**Comment by ibgreen** [comment]
Yes, the bump needs to go into both `master` and `4.1-release`. I don't think we need to urgently publish the patch release (we can wait a few days to see if any other fixes should be included), but it would be good to have the change landed on the branches..
